sql/hive/src/test/scala/org/apache/spark/sql/hive/test/TestHive.scala

```diff
@@ -588,20 +589,31 @@ private[hive] class TestHiveQueryExecution(
 
   override lazy val analyzed: LogicalPlan = {
     val describedTables = logical match {
-      case CacheTableCommand(tbl, _, _, _) => tbl.table :: Nil
+      case CacheTableCommand(tbl, _, _, _) => tbl :: Nil
       case _ => Nil
     }
 
     // Make sure any test tables referenced are loaded.
     val referencedTables =
       describedTables ++
-        logical.collect { case UnresolvedRelation(ident) => ident.last }
+        logical.collect { case UnresolvedRelation(ident) =>
+          TableIdentifier(ident.last, ident.init.lastOption)
```

Comment:
we should assert that `ident.length <= 2`

Reply:
Or you can just use:

```scala
import org.apache.spark.sql.connector.catalog.CatalogV2Implicits._
...
logical.collect { case UnresolvedRelation(ident) => ident.asTableIdentifier }
```

I guess.

Reply:
I see. this one should work.